common/web/gesture-recognizer/src/tools/build.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+#
+# Builds the include script for the current Keyman version.
+#
+
+# Exit on command failure and when using unset variables:
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# This script runs from its own folder
+cd "$(dirname $THIS_SCRIPT)"
```

Comment:
```suggestion
cd "$THIS_SCRIPT_PATH"
```

`$THIS_SCRIPT_PATH` is now always defined in build-utils.sh. Also `$THIS_SCRIPT_NAME` is `basename $THIS_SCRIPT`.